xml/art_sle_install_quick.xml

```diff
@@ -383,6 +383,12 @@
       selected automatically to fulfill its dependencies.
      </para>
      <para>
+      The registration server can mark some modules or extensions as
+      recommended. The recommended items are automatically preselected to
```

Comment:
Let's rephrase first sentence a little bit, e.g: "The registration server can mark some modules or extensions as      recommended depending on the product."